code/tsconfig.json

```diff
@@ -1,23 +1,23 @@
 {
   "compileOnSave": false,
   "compilerOptions": {
-    "ignoreDeprecations": "5.0",
+    "allowSyntheticDefaultImports": true,
     "baseUrl": ".",
-    "incremental": false,
-    "noImplicitAny": true,
+    "esModuleInterop": true,
     "forceConsistentCasingInFileNames": true,
+    "ignoreDeprecations": "5.0",
+    "incremental": false,
+    "isolatedModules": true,
     "jsx": "react",
-    "moduleResolution": "Node",
-    "target": "ES2020",
+    "lib": ["dom", "dom.iterable", "esnext"],
     "module": "CommonJS",
+    "moduleResolution": "Node",
+    "noImplicitAny": true,
+    "noUnusedLocals": false,
```

Comment:
Why did we change the `noUnusedLocals` setting?

Reply:
I changed it, because it's making CI fail over an unused variable.

Something that gets bundled out anyway.
It's just not something worth failing a whole build/compile over.

I'm interested in adding an eslint rule for it though.

Reply:
fair enough